src/main/groovy/be/cytomine/software/CheckingLoadSlurmProcessingServer.groovy

```diff
@@ -0,0 +1,259 @@
+package be.cytomine.software
+
+import be.cytomine.client.collections.Collection
+import be.cytomine.client.models.ProcessingServer
+import be.cytomine.software.communication.Communication
+import be.cytomine.software.communication.SSH
+import be.cytomine.software.consumer.Main
+import com.google.common.base.Splitter
+import com.google.common.collect.Lists
+
+import groovy.util.logging.Log4j
+import org.json.simple.JSONObject
+
+@Log4j
+class CheckingLoadSlurmProcessingServer {
```

Comment:
As it's supposed to be generic with respect to a processing method, where is the base class ?